client/src/components/events.js

```diff
@@ -14,14 +15,30 @@ function Events() {
             console.log('Events fetched...', events);
             });
           }, []);
+  
+          const postRequest = (newEvent) => {;
+            return fetch("http://localhost:8080/api/events", {
+              method: "POST",
+              headers: {"Content-Type": "application/json"},
+              body: JSON.stringify(newEvent),
+            })
+            .then((response) => {
+              return response.json();
+            })
+            .then((data) => {
+              setEvents((events) => [...events, data])
+            })
+          }
 
   return (
+    <div>
```

Comment:
The need to put a wrapping element in a component is so common that React has [Fragments](https://legacy.reactjs.org/docs/fragments.html) built specifically for this purpose. I would like to see you replace this `div` with a Fragment here in either the long or shortened form.